auction-service/.env.example

```
@@ -23,3 +23,8 @@ INTERNAL_API_SECRET=
 
 # Encryption
 ENCRYPTION_SECRET=
+
+# Service URLs (optional - defaults to localhost in local)
+USER_SERVICE_URL=
+TICKET_SERVICE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TICKET_SERVICE_URL key should go before the USER_SERVICE_URL key

(UnorderedKey)